tests/test_mariadb.py

```diff
@@ -276,3 +281,80 @@ def test_mariadb_healthcheck_galera_cluster_disabled(auto_container_per_test):
     _wait_for_server(conn)
 
     conn.run_expect([1], "healthcheck.sh --su-mysql --galera_online")
+
+
+_DB_ENV = {
+    "MARIADB_USER": _OTHER_DB_USER,
+    "MARIADB_PASSWORD": _OTHER_DB_PW,
+    "MARIADB_DATABASE": _TEST_DB,
+    "MARIADB_ROOT_PASSWORD": MARIADB_ROOT_PASSWORD,
+    "MARIADB_AUTO_UPGRADE": "1",
+}
+
+OS_VERSION = os.getenv("OS_VERSION", "")
+
+
+@pytest.mark.parametrize("ctr_image", MARIADB_CONTAINERS)
+@pytest.mark.skipif(
+    OS_VERSION in ("15.5",), reason="MariaDB upgrade scenario not supported"
+)
+def test_mariadb_upgrade(
+    container_runtime: OciRuntimeBase,
+    pytestconfig: pytest.Config,
+    ctr_image: DerivedContainer,
+    tmp_path: Path,
+) -> None:
+    mounts = [BindMount(host_path=tmp_path, container_path="/var/lib/mysql")]
+    mariadb_old = Container(
+        url="registry.opensuse.org/devel/bci/sle-15-sp6/containerfile/suse/mariadb:latest",
```

Comment:
please use the mariadb container from `registry.suse.com/suse/mariadb`, at a version that is different from the containers in `MARIADB_CONTAINER`.

Reply:
version 10.6 from `registry.suse.com/suse/mariadb` is added.